examples/gradle-kotlin-dsl/build.gradle.kts

```diff
@@ -1,5 +1,5 @@
 plugins {
-    id("com.saveourtool.diktat.diktat-gradle-plugin") version "1.2.5"
```

Comment:
I am testing the release on this example and as I can remember - latest gradle requires `settings.gradle.kts`. Can you please add it also? Empty one will be fine.

Reply:
already added